security-module/src/main/java/com/broadcom/apiml/library/service/security/service/security/config/InitializeTokenSecret.java

```diff
@@ -7,27 +7,25 @@
  *
  * Copyright Contributors to the Zowe Project.
  */
-package com.ca.mfaas.security.config;
-
-import com.ca.mfaas.product.web.HttpConfig;
-import com.ca.mfaas.security.token.TokenService;
+package com.broadcom.apiml.library.service.security.service.security.config;
```

Comment:
com.broadcom.apiml.library.service.security.service.security.config? Don't you think the package name is weirdly repeatable?